src/orders-v2/mocked-app-layout.tsx

```diff
@@ -0,0 +1,33 @@
+import { Box } from "@saleor/macaw-ui-next";
+import React from "react";
+
+export const MockedAppLayout = ({ children }: { children: React.ReactNode }) => {
```

Comment:
Why mocked? Is it used for storybook? Can we keep "mocked" stuff in some __tests__ or "storybook" folder?